examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -61,10 +61,10 @@
 # Define the dimension reduction to be used.
 # Here we use a classical univariate feature selection based on F-test,
 # namely Anova. We set the number of features to be selected to 500
```

Comment:
5% would be good enough I guess as we are selecting good features in the brain with 5%.

We wait and see if others are asked to change to 10%. Otherwise 5% is good enough.